cs3220-summer2017-lab3/src/lab3/CreateFoodAdminServlet.java

```diff
@@ -0,0 +1,58 @@
+package lab3;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import edu.csula.cs3220.examples.GuestBookEntry;
+
+/*A page to create an information of a food item*/
+
+@WebServlet("/admin/foods/create")
+public class CreateFoodAdminServlet extends HttpServlet {
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+		response.setContentType("text/html");
+		PrintWriter out = response.getWriter();
+		out.println("<link rel=\"stylesheet\" type=\"text/css\" href=\"../../app.css\">");
+		out.println("<title> Create Food Item </title>");
+		out.println("<h2> Create Food Item Form</h2><br>");
+
+		out.println("<form method=\"post\">");
+		out.println("<label>Food Name:</label><br>");
```

Comment:
Usually, when you use `label`, you want to have a `for` attribute pointing to the id of the input element so that you can click on the label to auto-focus the input element for better usability. e.g.

```
<label for="name">Food name:</label>
<input id="name" name="name" type="text">
```

will allow you to click on the food name to focus name input element.